builtin/credential/ldap/path_config.go

```diff
@@ -143,8 +166,22 @@ func (b *backend) pathConfigWrite(
 	if groupdn != "" {
 		cfg.GroupDN = groupdn
 	}
+	groupfilter := d.Get("groupfilter").(string)
```

Comment:
That logic was collapsed into the general case, and requires the following (different than default) configuration:

```
groupfilter: "(&(objectClass=person)(uid={{.Username}}))",
groupattr: "memberOf",
groupdn: "<search base for my org's user objects"
```

See the different scenarios I listed in the docs. I don't think the backend should try to be clever about this - they're really separate configurations.